libraries/decoder_midi/src/main/java/androidx/media3/decoder/midi/TrackEvent.java

```diff
@@ -139,10 +141,17 @@ public boolean populateFrom(ParsableByteArray parsableTrackEventBytes, int previ
           default: // Ignore all other Meta events.
             parsableTrackEventBytes.skipBytes(eventLength);
         }
-      } else {
+      } else if (firstByte == SYSEX_BEGIN_STATUS) {
```

Comment:
Thanks for the contribution.

Since you're here, can you change the if-else structure to be

```
if ((firstByte & 0xF0) != 0xF0) {
  ...
} else if (firstByte == META_EVENT_STATUS) {
  ...
} else if (firstByte == SYSEX_BEGIN_STATUS) { 
  ...
} else {
throw ParserException.createForUnsupportedContainerFeature(
            "Unknown track event: " + firstByte);
}